src/components/common/helpers/Helpers.tsx

```diff
@@ -1112,32 +1112,6 @@ export const reloadToastBody = () => {
     )
 }
 
-export function useHeightObserver(callback): [RefObject<HTMLDivElement>] {
```

Comment:
The contents of the custom hook seem a little convoluted and unnecessary to me. Since this is (was) being used only in TerminalView, I have refactored it out. React warns us from using useLayoutEffect anyways. The motive was probably that the actually resizing should be differed to the next repaint to prevent infinite loops in ResizeObserver callback. We shouldn't change the div size inside the resizeObserver. Thus using window.requestAnimationFrame will defer the actual resize until the next repaint saving us from the infinite resize.